android/src/main/java/tv/vizbee/rnsender/VizbeeNativeManager.java

```diff
@@ -167,6 +169,36 @@ public void disconnect() {
         }
     }
 
+    //----------------
+    // SignIn APIs
+    //----------------
+
+    @ReactMethod
+    public void onGetSignInInfo(ReadableMap vizbeeSignInInfoMap) {
+        Log.v(LOG_TAG, "Invoking onGetSignInInfo");
+        VizbeeSessionManager sessionManager = VizbeeContext.getInstance().getSessionManager();
+        if (null == sessionManager) {
+            Log.i(LOG_TAG, "sessionManager is null");
+            return;
+        }
+
+        VizbeeSession currentSession = sessionManager.getCurrentSession();
+        if (null == currentSession) {
+            Log.i(LOG_TAG, "currentSession is null");
+            return;
+        }
+
+        try {
+            JSONObject authInfo = RNJSONConverter.convertMapToJson(vizbeeSignInInfoMap);
+            JSONObject authJSONObject = new JSONObject();
+            authJSONObject.put("authInfo", authInfo);
+
+            currentSession.sendEventWithName("tv.vizbee.homesign.signin", authJSONObject); 
```

Comment:
Let's use contacts instead of magic numbers or string

Reply:
All magic strings are moved to VizbeeConstants file